pages/about.html

```diff
@@ -1,12 +1,34 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="about-D">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../style.css">
   <title>Document</title>
 </head>
 <body>
+  <header>
+ <hi class="about-header">Hola!</hi>
+</header>
+<main>
+  <img class="about-img"  src="/images/Screenshot (21).png" alt="Diana">
```

Comment:
Better to use relative paths.  Also `img` is a self-closing element.  Also in general for web servers it's a bad idea to use spaces in file names.  

```suggestion
  <img class="about-img"  src="../images/Screenshot (21).png" alt="Diana" />
```